### ISSUE TITLE
Undo/Redo commands in Edit menu do not work on macOS

### ISSUE THREAD

**Issue by Lars-Br** [ONLYOFFICE/DesktopEditors#388]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
The Undo and Redo commands in the Edit menu do nothing on macOS and for Redo, even the shortcut does not work.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Perform some action that can be undone. Then choose "Undo" from the "Edit" menu. Nothing happens. Then click the "Undo" button in the toolbar or press Command-Z. The action is undone. Now choose "Redo" from the "Edit" menu. Again nothing happens. Now press Command-Shift-Z. Not even the shortcut works. (Command-Y works, but that's not the correct shortcut on macOS and it's not the one that's displayed next to the Redo command in the menu.)  

**What is the expected behavior?**
The Undo / Redo commands should perform the same action as the corresponding toolbar buttons, and for Redo, Command-Shift-Z should work as well.

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
5.5.1 on macOS. I don't know if this used to work in previous versions.

**Comment by DenisDeeSign** [comment]
Hi, @Lars-Br.
I can confirm that the issue with edit menu is a bug 45400 in our internal tracker.
The Command-Shift-Z sequence is used by Pages and other Apple editors, but not MS Office which hotkeys we are implementing. We discuss it with our developers team.

**Comment by Lars-Br** [comment]
Thanks for the confirmation! Command-Shift-Z is mandated by the Apple Human Interface Guidelines for macOS, s. https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-bar-menus .

Why don't you implement both shortcuts? Command-Y is not assigned to anything in the Apple HIG, so could follow the HIG and still implement the Word shortcut as well.

**Comment by DenisDeeSign** [comment]
Yes, we agree that sequence is useful. Your suggestion has number 45435 in our internal tracker.
Thank you.

**Comment by HansJK** [comment]
Would love to see this feature!

**Comment by cluelessdev404** [comment]
This has been open for nearly 2 years and the issue's still here. Just commenting to acknowledge its continued existence.

**Comment by msr8** [comment]
Same, command+Y works but I couldn't find a mention of it anywhere except this thread. Also the GUI redo buttons dont work and show the shift+command+Z shortcut (in Edit>Redo)
![Screenshot 2022-12-25 at 10 00 20 AM](https://user-images.githubusercontent.com/79649185/209456985-ea32f9dc-ec54-4192-a08e-a098ce891faf.png)

**Comment by rafamel** [comment]
As this has been open since 2020, is there any plan to fix/implement this at some point within the year?

**Comment by Ampa** [comment]
Windows user here. Also used to using Ctrl + Shift + Z for redo.

Would dearly love to see this implemented.
